pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade

```diff
@@ -1,85 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We did move move this functionality somewhere else previously, right ?

Reply:
Partially, the logic is the same, but we don't show a dialog, just the progress messages. It is possible, that someone will want the dialog back, but the dialog will have to be rewritten anyway, because the logic is part of the Storage module now. Therefore, I suggest to remove this code for now and bring back the updated version if requested.

Reply:
I'm afraid this could be an issue in the future once this change lands in a RHEL release as I can image people expecting the dialog & not noticing this is now reported via a status message.